src/foam/nanos/alarming/Alarm.js

```diff
@@ -152,6 +152,7 @@ foam.CLASS({
     {
       name: 'stop',
       label: 'Stop Alarm',
+      availablePermissions: ['foam.nanos.alarming.Alarm.permission.stop'],
```

Comment:
Why the weird permission string.
Should this not be 
foam.nanos.alarming.alarm.rw.stop